src/toil/jobStores/aws/jobStore.py

```diff
@@ -827,7 +877,7 @@ def _bindDomain(self, domain_name, create=False, block=True):
         :param bool block: If False, return None if the domain doesn't exist. If True, wait until
                domain appears. This parameter is ignored if create is True.
 
-        :rtype: Domain|None
+        :rtype: str domain_name | None
         :raises SDBResponseError: If `block` is True and the domain still doesn't exist after the
```

Comment:
This sounds like it isn't true anymore; isn't `SDBResponseError` a Boto 2 thing?